### PR TITLE
Adding notes for tests that were moved

### PR DESCRIPTION
**Pull request by FlyNeopolitan**
Already sorted

### CODE REVIEW

**Comment by winglam** [comment]
Thanks for the notes! Can you please add them as specified by the [readme](https://github.com/TestingResearchIllinois/idoft#adding-notes)?

**Comment by winglam** [comment]
Thanks for the notes. Can you please document the notes as described in the [readme](https://github.com/TestingResearchIllinois/idoft#adding-notes)?
For tests with pretty much the same note (e.g., "In 32d5656ffd0aa4ba6caba830ac0205c7bf2609e3...") you can create just one issue for all of them.

**Comment by FlyNeopolitan** [comment]
> Thanks for the notes. Can you please document the notes as described in the [readme](https://github.com/TestingResearchIllinois/idoft#adding-notes)?
> For tests with pretty much the same note (e.g., "In 32d5656ffd0aa4ba6caba830ac0205c7bf2609e3...") you can create just one issue for all of them.



> Thanks for the notes. Can you please document the notes as described in the [readme](https://github.com/TestingResearchIllinois/idoft#adding-notes)?
> For tests with pretty much the same note (e.g., "In 32d5656ffd0aa4ba6caba830ac0205c7bf2609e3...") you can create just one issue for all of them.

Sorry I didn't see the old one. Thanks for your remind.

**Comment by winglam** [comment]
Thanks for creating the issues. Please update your changes with the link to the issues as described in the [readme](https://github.com/TestingResearchIllinois/idoft#adding-notes)

**Comment by FlyNeopolitan** [comment]
Hi, I have updated changes

File: pr-data.csv [hunk 1]
```diff
@@ -1670,18 +1670,18 @@ https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30a
...
+https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase4,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/30
+https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase6,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/31
+https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase8,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/32
+https://github.com/TooTallNate/Java-WebSocket,fa3909c391195178ccf5a92d4ac342a30ae247c8,.,org.java_websocket.protocols.ProtoclHandshakeRejectionTest.testHandshakeRejectionTestCase9,NOD,,,https://github.com/TestingResearchIllinois/idoft/issues/33
```
Comment (winglam) [inline pr-data.csv:1684]: Can you please double-check the debugging information links that you are adding? Some of these do not exist

**Review by winglam — COMMENTED** [review]
Thanks a lot for the changes, especially the update to the readme and the extra space character

File: pr-data.csv (hunk 1 above)
Comment (FlyNeopolitan) [reply pr-data.csv]: Sorry that's a big mistake. My fault. Now it's fixed

**Comment by winglam** [comment]
LGTM, thanks a lot for all of your changes!

**Pull request merged**